cpp/modmesh/buffer/ConcreteBuffer.hpp

```diff
@@ -168,9 +170,12 @@ class ConcreteBuffer
      *      Size of the memory buffer in bytes.
      */
     ConcreteBuffer(size_t nbytes, const ctor_passkey &)
-        : m_nbytes(nbytes)
+        : BufferBase<ConcreteBuffer>(nullptr, nullptr) //  initialize m_begin and m_end, but since we don't have the data yet, we set them to nullptr
```

Comment:
Don't delegate to the parent constructor setting `nullptr`.  The pointers will be overridden and explicitly initializing here wastes cycles.

Reply:
fixed.